news/_posts/2023-10-12-sc_subdomain.md

```diff
@@ -0,0 +1,19 @@
+layout: news
+---
+
+The European Galaxy Days 2023 CoFest combined the forces of administrator, developer and trainer to update and re-launch the [single cell](https://singlecell.usegalaxy.eu) Galaxy instance. Where previously there were two subdomains each with their own sets of tools, there is now a single subdomain with a suite of tools in an order that makes sense for users.
+
+The central page was updated for the first time in years, and some excellent workflow and training search widgets were created. This went hand-in-hand with the creation of the [Galaxy Single Cell Community of Practice page](https://galaxyproject.org/projects/singlecell/).
+This infrastructure connects - for the first time - single cell users from novices performing training in Galaxy, to searching more advanced workflows and tools beyond the training, and finally able to contribute, request tools, provide feedback or join the Community of Practice.
+
+#What spawned this project?
+"Trying to navigate users through all the quirks of Galaxy when they tried to move outside of the tutorials was challenging, and having to explain why we had two sites that both focused on single cell analysis was also tricky. This subdomain makes it a lot easier to communicate the potential to users - they can immediately see how many different toolsuites are available without being overwhelmed. They don't have to debate whether to learn R or Python, Seurat or Scanpy - they can just try both! Plus, data formats is the thing that stumps users more than anything. My group was constantly losing track of which tool could do what to convert data formats, and I am absolutely thrilled to have an entire section devoted to these random tools that are so vital to applying the analysis to public or private data."
+
+#What was it like to do this at EGD 2023 CoFest?
+"I had no idea how difficult it would be to make what seemed like quite superficial changes to make the tool panels more usable! José was an absolute star, navigating through all of this seamlessly. Pavan kept finding single cell tools under a ton of different metadata titles, so it was also exciting to see just how many different tools were out there and to get them all in the same place. There's so much more to play with than I realised! Helena jumped in virtually to make these banging widgets to prevent the sites from becoming outdated, and give contributors more of a voice in the site - if anyone shares a workflow tagged #singlecell on any of the Big 3 public instances, it's now on our single cell page. I'm so glad we had the CoFest though, because getting together all the different single cell sites, getting all the tools that we found in the right spot, it required the entire day and then some and a lot of collaboration. At one point I was literally highlighting tool IDs in a giant list. It just wouldn't have been feasible virtually. I'm so excited for the next training session to just see how users navigate the new site."
```

Comment:
```suggestion

> I had no idea how difficult it would be to make what seemed like quite superficial changes to make the tool panels more usable! José was an absolute star, navigating through all of this seamlessly. Pavan kept finding single cell tools under a ton of different metadata titles, so it was also exciting to see just how many different tools were out there and to get them all in the same place. There's so much more to play with than I realised! Helena jumped in virtually to make these banging widgets to prevent the sites from becoming outdated, and give contributors more of a voice in the site - if anyone shares a workflow tagged #singlecell on any of the Big 3 public instances, it's now on our single cell page. I'm so glad we had the CoFest though, because getting together all the different single cell sites, getting all the tools that we found in the right spot, it required the entire day and then some and a lot of collaboration. At one point I was literally highlighting tool IDs in a giant list. It just wouldn't have been feasible virtually. I'm so excited for the next training session to just see how users navigate the new site.
{: .quote}
```

if you want to use a semantic quote. (Do these quotes have a speaker that should be credited?)

Reply:
Yes, me! Lol